### PR TITLE
Added grid

### PR DESCRIPTION
**Pull request by nishasen**
Added 2 items and 3 items in a grid
grids

### CODE REVIEW

File: Stylesheet/Components/grid.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* This is grid file */
+
+.grid2 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (coldpigli) [inline Stylesheet/Components/grid.css]: liked how your used repeat function instead of writing 1fr,1fr.

**Review by coldpigli — APPROVED** [review]
Good to merge. Everything checks okay

File: Stylesheet/Components/grid.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    grid-gap: 0.5rem;
```
Comment (srejitk) [inline Stylesheet/Components/grid.css]: You can keep the **display:grid** and **grid-gap:0.5rem** in another class for better modularity

**Review by srejitk — COMMENTED** [review]
Suggesting Minor changes

File: Stylesheet/Components/grid.css (hunk 2 above)
Comment (nishasen) [reply Stylesheet/Components/grid.css]: Sure changing that, thank you

File: Stylesheet/Components/grid.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/Components/grid.css]: Thank you

**Pull request merged**